File: Application/Backend/InventoryManagementSystem.Model/DTO/UserDTO.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace InventoryManagementSystem.Model
{
    public class UserDTO
    {
        public int UserId { get; set; }
        public string FirstName { get; set; }
        public string LastName { get; set; }
        public string UserName { get; set; }
        public string Email { get; set; }
        public int RoleId { get; set; }
        public string RoleName { get; set; }

        public string LastAction {  get; set; }
        public int TotalLogs { get; set; }
        public DateTime? LastActivity { get; set; }
    }
}

File: Application/Frontend/inventorymanagemetnsystem/src/app/models/UserDTO.ts
export interface UserDTO {
  userId: number;
  firstName: string;
  lastName: string;
  userName: string;
  email: string;
  roleId: number;
  roleName: string;
  profileImagePath: string;
  lastAction: string;
  totalLogs: number;
  lastActivity: Date;
}
